### PR TITLE
Adjust alert/aside design

### PR DESCRIPTION
**Pull request by parlough**
Makes some adjustments to:

- More closely match developer's expectations from GitHub alerts
- Reduce spacing for to better align with upcoming changes to dart.dev's alerts
- Increase contrast of alert text in dark mode
- Add a unique icon and color for each type

**Dark mode:**

New dark mode styles

**Light mode:**

New light mode styles

### CODE REVIEW

**Comment by parlough** [comment]
\cc @devoncarew

File: lib/src/generator/templates.aot_renderers_for_html.dart [hunk 1]
```diff
@@ -3531,7 +3531,7 @@ String _deduplicated_lib_templates_html__head_html(TemplateDataBase context0) {
   buffer.write('''
   <link rel="preconnect" href="https://fonts.gstatic.com">
   <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:ital,wght@0,300;0,400;0,500;0,700;1,400&display=swap" rel="stylesheet">
-  <link href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@24,400,0,0" rel="stylesheet">
```
[the remark this answers is not in the document]
Comment (parlough) [reply lib/src/generator/templates.aot_renderers_for_html.dart:3534]: I should have explained this, sorry. But we mostly use a size closer to 24px for icons, so it makes sense to specify that as the default optical size. Seemed to look better for the other icons as well.

[the remark this answers is not in the document]
Comment (devoncarew) [reply lib/src/generator/templates.aot_renderers_for_html.dart:3534]: Yeah, no, this is a good way to get the defaults; I wasn't aware that the URL had been requesting the initial font size.

**Review by devoncarew — APPROVED** [review]
> I'm happy to make changes though (here and on dart.dev)! I don't feel tied to any design or style of these as the important change here is the improved contrast. Whatever we end up with here, I'll try to align with that on dart.dev :)

I feel like you guys should own the styles, and general alignment between dartdoc and dart.dev. lgtm :)

**Review by srawlins — APPROVED** [review]
Love it!

**Comment by parlough** [comment]
Thanks both of you for the review!

Feel free to merge when ready. I don't have access on this repository.

**Pull request merged**